src/UX/Identification/IdentificationFactory.php

```diff
@@ -0,0 +1,17 @@
+<?php namespace ILIAS\UX\Identification;
+
+/**
+ * Class IdentificationFactory
+ *
+ * @author Fabian Schmid <fs@studer-raimann.ch>
+ */
+class IdentificationFactory {
+
+	/**
+	 * @inheritdoc
```

Comment:
Inherit from where? There is not interface for this factory (which I am not very fond about).